entrypoint.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 echo running script
-poetry run python auto_post_classifier/main.py -d data/AntiIsraeli.csv -n 5 --no-api
\ No newline at end of file
+poetry run python auto_post_classifier/main.py -d data/AntiIsraeli.csv --no-shuffle --no-api -n -1
```

Comment:
@nilnil47 I am assuming that the `--no-shuffle` is here because in this mode of operation the *whole* input should be processed. Is that correct?